frappe/email/test_email_body.py

```diff
@@ -136,6 +137,20 @@ def test_get_email_header(self):
 		html = get_header('This is string')
 		self.assertTrue('<span>This is string</span>' in html)
 
+	def test_8bit_utf_8_decoding(self):
+		text_content_bytes = b"\xed\x95\x9c\xea\xb8\x80\xe1\xa5\xa1\xe2\x95\xa5\xe0\xba\xaa\xe0\xa4\x8f"
+		text_content = text_content_bytes.decode('utf-8')
+
+		content_bytes = b"""MIME-Version: 1.0
+Content-Type: text/plain; charset=utf-8
```

Comment:
needs indentation.

Reply:
Please be more speciffic which indentation to use. There are multiple patterns in test_email_body.py including this one. Please see [this](https://github.com/frappe/frappe/blob/26197ee29edd7ae0905fed0bc61b2d866294640f/frappe/email/test_email_body.py#L100-L108).

Reply:
I guess just let it be for now.